libraries/redcore/database/driver.php

```diff
@@ -113,7 +111,7 @@ public static function getInstance($options = array())
 				// Create our new JDatabase connector based on the options given.
 				try
 				{
-					$instance = new $class($options);
+					$instance = $class::getInstance($options);
```

Comment:
I wanted to get new instance of the object since it references to a different JDriver...